clang/include/clang/Basic/Attr.td

```diff
@@ -559,12 +559,12 @@ def AbiTag : Attr {
 }
 
 def CMSThreadSafe : InheritableAttr {
-  let Spellings = [ CXX11<"cms","thread_safe", 200809>, C2x<"cms", "thread_safe> ];
+  let Spellings = [ CXX11<"cms","thread_safe", 200809>, C2x<"cms", "thread_safe"> ];
```

Comment:
One other suggestion from the documentation on adding attributes is that the namespace should be "clang" if it will be use by the static analyzer. Please try 
 let Spellings = [ CXX11<"clang","thread_safe", 200809>, C2x<"clang", "thread_safe"> ];

Reply:
Oh wait this will require a coordinated change to the definition of the CMS_THREAD_SAFE(sp?) macro.